Rakefile

```diff
@@ -5,4 +5,5 @@ Yast::Tasks.submit_to :sle12sp1
 Yast::Tasks.configuration do |conf|
   #lets ignore license check for now
   conf.skip_license_check << /.*/
+  conf.obs_sr_project = "SUSE:SLE-12-SP1:Update"
```

Comment:
OK, so at some point ` Yast::Tasks.submit_to :sle12sp1` will be switched from GA to Update, and the explicit `conf.obs_sr_project` is to make it independent of that.
LGTM.